src/components/HomepageCards/index.js

```diff
@@ -0,0 +1,280 @@
+import React from 'react';
+import clsx from 'clsx';
+import styles from './styles.module.css';
+import { useColorMode } from '@docusaurus/theme-common';
+
+const FeatureList = [
+  {
+    title: 'Seamless workflow from planning to commanding',
+    Img: require('@site/static/img/cards/aerie-feature-planning-commanding-light.png').default,
+    ImgDark: require('@site/static/img/cards/aerie-feature-planning-commanding-dark.png').default,
+    size: 'col--8',
+    description: (
+      <>
+        Build sequences and commands that execute on-board a spacecraft directly from activities in your plans. Author
+        and refine sequences derived from your command dictionary.
+      </>
+    ),
+  },
+  {
+    title: 'Extensible mission modeling and simulation',
+    Img: require('@site/static/img/cards/aerie-feature-mission-modeling-light.png').default,
+    ImgDark: require('@site/static/img/cards/aerie-feature-mission-modeling-dark.png').default,
+    size: 'col--4',
+    description: (
+      <>
+        Model your system with access to the entire Java ecosystem. Validate plans against your models using Aerie's
+        discrete event simulator using the user interface or Aerie API.
```

Comment:
Minor, ignorable: Double use of `using` in this sentence. Maybe replace one of them with something like "through" or "via"?

Reply:
switched to through